client/components/locations/dashboard/_index.html

```diff
@@ -21,7 +21,7 @@
     <div flex flex-gt-sm="{{fs.panel ? 100 : 100}}" layout="column">
       <div flex flex-gt-sm="{{fs.panel ? 100 : 100}}" layout="column" class="dashboard-charts">
         <div layout="" layout-wrap>
-          <div flex="100" flex-gt-md="{{fs.panel ? 100 : 33}}" iiing-hide="fs && fs.panel != 'usage'">
+          <div flex="100" flex-gt-sm="{{fs.panel ? 100 : 33}}" iiing-hide="fs && fs.panel != 'usage'">
```

Comment:
@mep253 What was the reasoning behind altering these? @ZakMooney Can you double check this is ok with your latest changes - might be different on master.